src/migration-stub.ts

```diff
@@ -0,0 +1,29 @@
+function migrationStub(id: string, coffeeScript: boolean): string {
+  if (coffeeScript) {
+    return `module.exports.id = "${id}"
```

Comment:
```suggestion
    return module.exports.id = "${id}"
```

This weird quotation mark comments out all the module functions... um... should we look at this file a bit more in depth?

But then again, maybe we don't need to even have this file, if coffeeScript is removed.

Reply:
Yeah, this function is returning a string and the coffee part can go. I remember removing the coffeescript part but I missed it in this PR, it must be in one of the later ones.